spark_pipeline_framework/transformers/framework_csv_loader/v1/framework_csv_loader.py

```diff
@@ -20,6 +20,7 @@ def __init__(
         path_to_csv: Union[str, List[str], Path],
         delimiter: str = ",",
         has_header: bool = True,
+        clean_column_names: bool = True,
```

Comment:
@imranq2 Note that I defaulted these to True because it will save us some code when we consume it, but this is technically a breaking change that _should_ require a major version bump. If you'd rather us only due a minor version bump for release, I can change this to default to False. Let me know what you think.

Reply:
if there is a default then it would be backwards compatible